integration/query_set_test.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2021 FerretDB Set.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type testCase struct {
+	id     string
+	update bson.D
+	result bson.D
+	err    *mongo.WriteError
+	stat   *mongo.UpdateResult
+	alt    string
+}
+
+func TestSetOperatorOnString(t *testing.T) {
+	t.Parallel()
+
+	for name, tc := range map[string]testCase{
+		"SetMany": {
```

Comment:
We already have `Set` in the test name so it's ok to skip it in the test case name. It's just easier to read test cases' names without it.

Reply:
changed.